File: ELearningFrontEnd/src/app/dtos/review/review.req.ts
export class ReviewReq{

    comment:string;
    rating:number;
    courseId:number;
    userId:number;

    constructor(data: any) {
        this.comment = data.comment;
        this.rating = data.rating;
        this.courseId = data.courseId;
        this.userId = data.userId
    }
}

File: ELearningBackEnd/src/main/java/com/example/elearningbackend/review/ReviewReq.java
package com.example.elearningbackend.review;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class ReviewReq {

    private String comment;

    private int rating;

    private long courseId;

    private long userId;

    private boolean isActivated;
}
